python-qa-tools/load-test/send_load.py

```diff
@@ -0,0 +1,37 @@
+import argparse, os
+import json
+import sys
+from core.keys import keys_show
+from modules.auth.query import query_account
+from modules.bank.tx import tx_send
+from utils.commands import exec_command
+from utils.types import account_type
+
+# import env values
+DAEMON = os.getenv('DAEMON')
+DENOM = os.getenv('DENOM')
+CHAINID = os.getenv('CHAINID')
+DAEMON_HOME = os.getenv('DAEMON_HOME')
+
+parser = argparse.ArgumentParser(description='This program takes inputs for intializing multi message load test.')
+parser.add_argument('-s', '--sender', type= account_type, default = keys_show("account1")[1]['address'],help= 'From which account the transaction should be intialized')
+parser.add_argument('-r','reciever', type= account_type, default = keys_show("account1")[1]['address'], help= 'Reciever account number.')
+args = parser.parse_args()
+
+acc1, acc2 = args.sender, args.reciever
```

Comment:
```suggestion
acc1, acc2 = args.sender, args.receiver
```

Reply:
I feel it's better to define variables as `sender`, `receiver` or `from`.`to` instead of `acc1`,`acc2` for code readability

Reply:
We are making Vise versa transactions(acc1->acc2 and acc2->acc1). I think we don't have the scope here to fix a specific Name to this.